test/ICSharpCode.SharpZipLib.Tests/Zip/ZipEncryptionHandling.cs

```diff
@@ -72,6 +72,44 @@ public void ZipFileAesDecryption()
 			}
 		}
 
+		/// <summary>
+		/// Tests for reading encrypted entries using ZipInputStream.
+		/// </summary>
+		/// <param name="aesKeySize"></param>
+		[Test]
+		[Category("Encryption")]
+		[Category("Zip")]
+		[TestCase(0, CompressionMethod.Deflated)]
+		[TestCase(0, CompressionMethod.Stored)]
+		[TestCase(128, CompressionMethod.Deflated)]
+		[TestCase(128, CompressionMethod.Stored)]
+		[TestCase(256, CompressionMethod.Deflated)]
+		[TestCase(256, CompressionMethod.Stored)]
+		public void ZipInputStreamDecryption(int aesKeySize, CompressionMethod compressionMethod)
+		{
+			var password = "password";
+
+			using (var ms = new MemoryStream())
+			{
+				WriteEncryptedZipToStream(ms, password, aesKeySize, compressionMethod);
+				ms.Seek(0, SeekOrigin.Begin);
+
+				using (var zis = new ZipInputStream(ms))
+				{
+					zis.IsStreamOwner = false;
+					zis.Password = password;
+
+					var hmm = zis.GetNextEntry();
```

Comment:
Codacy found an issue: [Remove the unused local variable 'hmm'.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745488)